tests/test_water.py

```diff
@@ -28,38 +30,49 @@ def test_water_absorption():
                         6.09432615e-01, 4.28359939e-01, 3.24924372e-01, 2.54184653e-01,
                         2.02567146e-01, 1.68736041e-01, 1.10857634e+00, 6.99603767e-01,
                         4.91739726e-01, 3.72999916e-01, 2.91793607e-01, 2.32538816e-01,
-                        1.93702088e-01]
-    input_temps = [0, 10, 20, 30, 40, 50, 60]
-    input_fs = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
-    idx = 0
+                        1.93702088e-01])
+    input_temps = np.array([0, 10, 20, 30, 40, 50, 60], dtype=float)
+    input_fs = np.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15], dtype=float)
+    idx : int = 0
     for f in input_fs:
         for temp in input_temps:
             assert abs(expected_outputs[idx] - water_absorption(f,
                                                                 temp)) < 1e-6, "Expected value deviates from expected " \
                                                                                "absorption value "
```

Comment:
This is not introduced in this PR but I think readability can be improved here by:
1. Converting nested loop to single level loop using `itertools.product`
2. Assigning function output to a variable before the comparison
These two steps should help to reduce max line length too.